crypto/bn256/google/constants.go

```diff
@@ -13,13 +13,14 @@ func bigFromBase10(s string) *big.Int {
 	return n
 }
 
-// u is the BN parameter that determines the prime: 1868033³.
+// u is the BN parameter that determines the prime.
 var u = bigFromBase10("4965661367192848881")
 
-// p is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
+// P is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
 var P = bigFromBase10("21888242871839275222246405745257275088696311157297823662689037894645226208583")
 
 // Order is the number of elements in both G₁ and G₂: 36u⁴+36u³+18u²+6u+1.
+// Order - 1 = 2^28 * 3^2 * 13 * 29 * 983 * 11003 * 237073 * 405928799 * 1670836401704629 * 13818364434197438864469338081.
```

Comment:
This doesn't match..? 
```
>>> 2^28 * 3^2 * 13 * 29 * 983 * 11003 * 237073 * 405928799 * 1670836401704629 * 13818364434197438864469338081 
18120013071297999241489839011809292012244764632993302696031169913432
```

Reply:
https://www.wolframalpha.com/input/?i=2%5E28+*+3%5E2+*+13+*+29+*+983+*+11003+*+237073+*+405928799+*+1670836401704629+*+13818364434197438864469338081

Reply:
Or right. Python uses `**` not `^`. 
```
>>> 2**28 * 3**2 * 13 * 29 * 983 * 11003 * 237073 * 405928799 * 1670836401704629 * 13818364434197438864469338081 
21888242871839275222246405745257275088548364400416034343698204186575808495616
```